tests/Tests.FeatureManagement/appsettings.json

```diff
@@ -63,6 +63,135 @@
           }
         }
       ]
+    },
+    "VariantFeature": {
+      "Assigner": "Test",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "V1",
+          "ConfigurationReference": "Ref1"
+        },
+        {
+          "Name": "V2",
+          "ConfigurationReference": "Ref2",
+          "AssignmentParameters": {
+            "P1": "V1"
+          }
+        }
+      ]
+    },
+    "ContextualVariantFeature": {
+      "Assigner": "ContextualTest",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "V1",
+          "ConfigurationReference": "Ref1"
+        },
+        {
+          "Name": "V2",
+          "ConfigurationReference": "Ref2",
+          "AssignmentParameters": {
+            "AllowedAccounts": [
+              "abc"
+            ]
+          }
+        }
+      ]
+    },
+    "ContextualVariantTargetingFeature": {
+      "Assigner": "Targeting",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "V1",
+          "ConfigurationReference": "Ref1"
+        },
+        {
+          "Name": "V2",
+          "ConfigurationReference": "Ref2",
+          "AssignmentParameters": {
+            "Audience": {
+              "Users": [
+                "Jeff"
+              ]
+            }
+          }
+        }
+      ]
+    },
+    "AccumulatedTargetingFeature": {
+      "Assigner": "Targeting",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "V1",
+          "ConfigurationReference": "Ref1"
+        },
+        {
+          "Name": "V2",
+          "ConfigurationReference": "Ref2",
+          "AssignmentParameters": {
+            "Audience": {
+              "DefaultRolloutPercentage": 33
+            }
+          }
+        },
+        {
+          "Name": "V3",
+          "ConfigurationReference": "Ref3",
+          "AssignmentParameters": {
+            "Audience": {
+              "DefaultRolloutPercentage": 33
+            }
+          }
+        }
+      ]
+    },
+    "AccumulatedGroupsTargetingFeature": {
+      "Assigner": "Targeting",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "V1",
+          "ConfigurationReference": "Ref1"
+        },
+        {
+          "Name": "V2",
+          "ConfigurationReference": "Ref2",
+          "AssignmentParameters": {
+            "Audience": {
+              "Groups": [
+                {
+                  "Name": "r",
+                  "RolloutPercentage": 33
```

Comment:
It will be interesting to set different percentage for each variant and validate it is the case. While the default variant is defined at the first, it has the least priority for variant assignment.

Reply:
I updated the test to use 3 different percentages. 15, 35, and 50.